core/cloud_connected_optimizer.py

```diff
@@ -0,0 +1,50 @@
+import logging
+import time
+
+from core.utils.enums 					import ActionableEventTypes, RawEventTypes, MessageEventTypes
+from core.utils.enums 					import JobTypes, callback_handler_no_default
+from core.utils.dmljob 					import (DMLJob, DMLAverageJob, DMLCommunicateJob, DMLServerJob, \
+												DMLInitializeJob, DMLSplitJob, DMLTrainJob, DMLValidateJob)
+from core.utils.keras 					import serialize_weights, deserialize_weights
+from core.utils.dmlresult 				import DMLResult
+from core.blockchain.blockchain_utils	import TxEnum, content_to_ipfs
+from core.fed_avg_optimizer import FederatedAveragingOptimizer
+
+
+class CloudConnectedOptimizer(FederatedAveragingOptimizer):
```

Comment:
1. Class docstring?

2. Can you summarize all of the differences that this class brings? This can be put in the docstring if you want.

Reply:
Truuuu. Will do.

Reply:
Done